### PR TITLE
Editable classifications

### PR DESCRIPTION
**Pull request by alexmcode**
This would be a first version for classifications edit form. Saving the form doesn't do anything at this point. Until we have a backend function to handle this should we do something more in this PR?

### CODE REVIEW

**Comment by smcgregor** [comment]
Can you add a function to [here](https://github.com/PartnershipOnAI/aiid/blob/master/realm.md)? I'll drop it into the backend and test it with your deploy preview.

**Comment by smcgregor** [comment]
Looking good, remaining items are:

* Add the ability to create classifications for incident records according to taxonomies in the taxa collection
* Add support for multiple taxonomies displaying on the citation page to edit
* Add Realm function for editing the taxonomy to the [documentation for Realm functions](https://github.com/PartnershipOnAI/aiid/blob/master/realm.md). I'll add and test it on the backend.

**Comment by smcgregor** [comment]
@alexmcode `Add Realm function for editing the taxonomy to the documentation for Realm functions. I'll add and test it on the backend.` is blocking merge here. The other two items can be on a separate pull request.

**Comment by alexmcode** [comment]
@smcgregor the first 2 points were already done. Just needed to double-check the backend function. All is needed is testing

**Comment by smcgregor** [comment]
I created the realm function per this pull request and will merge shortly.

File: site/gatsby-site/src/components/TaxonomyForm.js [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+
+      try {
+        runQuery(
+          {
+            namespace,
+          },
+          (res) => {
+            resolve(res);
+          },
+          'mongodb-atlas',
```
Comment (smcgregor) [inline site/gatsby-site/src/components/TaxonomyForm.js:89]: These should be pulled from site config, not inlined.

File: site/gatsby-site/src/components/TaxonomyForm.js [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+        runQuery(
+          {
+            // 'classifications.Publish': true,
+            incident_id: incidentId,
+            namespace,
+          },
+          (res) => {
+            resolve(res);
+          },
+          'mongodb-atlas',
```
Comment (smcgregor) [inline site/gatsby-site/src/components/TaxonomyForm.js:113]: Same here.

**Comment by smcgregor** [comment]
@alexmcode I left two inline comments about moving the query config out of the component. I am going to merge this now, but please address that comment in a follow up pull request.

**Review by smcgregor — APPROVED** [review]

**Pull request merged**